packages/bootleg-tokens/contracts/ISharedRoyaltyTokenEnumerable.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.5.3;
+
+/**
+ * @title Shared Royalty NFT Enumerable interface for franchisors
+ */
+contract ISharedRoyaltyTokenEnumerable {
+
+    function totalSupply() public view returns (uint256);
+    function tokenOfOwnerByIndex(address owner, uint256 index) public view returns (uint256 tokenId);
+
+    function tokenByIndex(uint256 index) public view returns (uint256);
+
+    /**
+    * @notice Gets the total number of Franchisees given a Token ID
+    * @param _tokenId The identifier of the NFT
+    * @return uint256 The number of franchisees
+    */
+    function tokenFranchiseesTotal(uint256 _tokenId) public view returns (uint256 count);

```

Comment:
`s/Franchisees/Franchisors/` everywhere

Reply:
Oh, okay. again, Followed the example code in the Mural. Will correct.